.github/workflows/pr-check.yaml

```diff
@@ -62,8 +62,23 @@ jobs:
 
       - uses: actions/upload-artifact@v3.1.2
         with:
-          name: windows-exe
-          path: ./dist/podman-desktop*.exe
+          name: windows-installer-x64
+          path: ./dist/podman-desktop*-setup-x64.exe
+
+      - uses: actions/upload-artifact@v3.1.2
+        with:
+          name: windows-installer-arm64
+          path: ./dist/podman-desktop*-setup-arm64.exe
+
+      - uses: actions/upload-artifact@v3.1.2
+        with:
+          name: windows-exe-x64
+          path: ./dist/podman-desktop*-next-x64.exe
```

Comment:
Shouldn't it be `./dist/podman-desktop*-x64.exe` without `next` as in the portable artifactName there is no next?

Reply:
I wanted to check with the PR

but on my end I have in dist folder

```
ls -l dist
-rw-r--r--@  1 benoitf  staff       7831 Aug 23 10:14 builder-debug.yml
-rw-r--r--@  1 benoitf  staff       1888 Aug 23 10:12 builder-effective-config.yaml
-rw-r--r--@  1 benoitf  staff        713 Aug 23 10:14 latest.yml
-rw-r--r--@  1 benoitf  staff   83757463 Aug 23 10:14 podman-desktop-1.4.0-next-arm64.exe
-rw-r--r--@  1 benoitf  staff   83930056 Aug 23 10:14 podman-desktop-1.4.0-next-setup-arm64.exe
-rw-r--r--@  1 benoitf  staff      88850 Aug 23 10:14 podman-desktop-1.4.0-next-setup-arm64.exe.blockmap
-rw-r--r--@  1 benoitf  staff  231126266 Aug 23 10:14 podman-desktop-1.4.0-next-setup-x64.exe
-rw-r--r--@  1 benoitf  staff     240231 Aug 23 10:14 podman-desktop-1.4.0-next-setup-x64.exe.blockmap
-rw-r--r--@  1 benoitf  staff  314515309 Aug 23 10:14 podman-desktop-1.4.0-next-setup.exe
-rw-r--r--@  1 benoitf  staff     327191 Aug 23 10:14 podman-desktop-1.4.0-next-setup.exe.blockmap
-rw-r--r--@  1 benoitf  staff  230953372 Aug 23 10:14 podman-desktop-1.4.0-next-x64.exe
-rw-r--r--@  1 benoitf  staff  314342680 Aug 23 10:14 podman-desktop-1.4.0-next.exe
```

so if I pickup `./dist/podman-desktop*-x64.exe` it means it'll take both the installer and the portable file

Reply:
on PR side it looks OK

![image](https://github.com/containers/podman-desktop/assets/436777/e2392f7b-72bd-4c8b-a22d-2a8b4afbe04d)